.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           fetch-depth: 0
```

Comment:
@arnaud-moncel @jeffladiray I'm pretty sure you could have useful insights / comments on what I've changed. Otherwise, if you could point me to someone helpful, that would be awesome ❤️